README.md

```diff
@@ -95,185 +95,14 @@ Other options include:
 
 Currently, the CLI supports validation of OpenAPI documents and lints them based on our default ruleset. It does not support custom rulesets at this time. Although if you want to build and run custom rulesets outside of the CLI, see [Customization](#Customization).
 
-### Example: Linting an OpenAPI document
-
-Spectral includes a number of ready made rules and functions for OpenAPI v2 and v3 documents.
-
-This example uses the OpenAPI v3 rules to lint a document.
-
-```javascript
-const { Spectral } = require('@stoplight/spectral');
-const { oas3Functions, oas3Rules } = require('@stoplight/spectral/rulesets/oas3');
-// for YAML
-const { parseWithPointers } = require("@stoplight/yaml");
-const myOAS = parseWithPointers(`
-responses:
-  '200':
-    description: ''
-    schema:
-      $ref: '#/definitions/error-response'
-`)
-
-// an OASv3 document
-const myOAS = {
-  // ... properties in your document
-  responses: {
-    '200': {
-      description: '',
-      schema: {
-        $ref: '#/definitions/error-response',
-      },
-    },
-  },
-  // ... properties in your document
-};
-
-// create a new instance of spectral with all of the baked in rulesets
-const spectral = new Spectral();
-
-spectral.addFunctions(oas3Functions());
-spectral.addRules(oas3Rules());
-
-spectral.addRules({
-  // .. extend with your own custom rules
-});
-
-// run!
-spectral.run(myOAS).then(results => {
-  console.log(JSON.stringify(results, null, 4));
-});
-```
-
-You can also [add to these rules](#Creating-a-custom-rule) to create a customized linting style guide for your OpenAPI documents.
+## Programmatic usage
 
-The existing OAS rules are opinionated. There might be some rules that you prefer to change. We encourage you to create your rules to fit your use case. We welcome additions to the existing rulesets as well!
+Should you need to use Spectral as a library feel free to check out the ["Using Spectral in code"](docs/js-api.md) documentation.
 
 ## Rulesets
 
 You can find all about [rulesets here](docs/rulesets.md).
 
-## Advanced
-
-### Customization
-
-There are three key concepts in Spectral: **Rulesets**, **Rules** and **Functions**.
```

Comment:
Yeah lets keep these under "## Concepts" and then move the creating and editing stuff elsewhere.